crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -32,10 +32,21 @@ impl ClassInfo {
         None
     }
 
-    /// Compare based on variants weight. Classes with higher weight go first.
+    /// Compare based on variants weight. Classes with lower weight go first.
+    /// First compare variants weight length. Only if their equal compare their actual weight.
```

Comment:
Is this really faster? We should benchmark because I'm not sure.

Reply:
Sorry, I probably didn't explain it well enough. I resorted to this because I tried by letting the native `bitvec` `cmp` method to handle comparison but it didn't work as I expected. It seems it doesn't check the `BitVec`s length first to establish which one is greater than the other.
I don't actually know what it does under the hood, I haven't checked yet.
I'm not well versed enough with Rust to think about optimizations first, my solution wasn't about speed 😁